templates/out-of-sync.j2

```diff
@@ -106,9 +107,17 @@ $(document).ready(function () {
                 <span class="text-success" title="added">+{{added}}</span>&nbsp;<span class="text-danger" title="removed">-{{removed}}</span>
             </td>
             <td>{{ d.dependent_depth }}</td>
+            <td data-order="{{d.status.mathlib4_sync_prs | length}}">
+                <ul>
+                    {%- for pr in d.status.mathlib4_sync_prs %}
+                        <li><a href="https://github.com/leanprover-community/mathlib4/pull/{{pr.num}}"
+                            title="{%- for label in pr.labels %}{{label.name}}{{ ", " if not loop.last }}{%- endfor %}">#{{pr.num}}</a></li>
+                    {%- endfor %}
+                </ul>
```

Comment:
Given we expect to rarely have more than one, I think it would be better to just use separate lines
```suggestion
                {% set separator = joiner('<br />') %}
                {%- for pr in d.status.mathlib4_sync_prs %}{{ separator() }}
                    <a href="https://github.com/leanprover-community/mathlib4/pull/{{pr.num}}"
                        title="{{ pr.labels | map(attribute="name") | join(", ") }}">#{{pr.num}}</a>
                 {%- endfor %}
```